airbyte-integrations/connectors/source-google-analytics-data-api/integration_tests/configured_catalog.json

```diff
@@ -91,30 +98,33 @@
         "supported_sync_modes": ["full_refresh", "incremental"],
         "source_defined_cursor": true,
         "default_cursor_field": ["date"],
-        "source_defined_primary_key": [["uuid"]]
+        "source_defined_primary_key": [["property_id"], ["date"]]
       },
-      "sync_mode": "full_refresh",
-      "destination_sync_mode": "overwrite"
+      "sync_mode": "incremental",
+      "destination_sync_mode": "overwrite",
+      "primary_key": [["property_id"], ["date"]]
     },
     {
       "stream": {
         "name": "cohort_report",
         "json_schema": {},
         "supported_sync_modes": ["full_refresh"],
-        "source_defined_primary_key": [["uuid"]]
+        "source_defined_primary_key": [["property_id"], ["cohort"], ["cohortNthDay"]]
       },
       "sync_mode": "full_refresh",
-      "destination_sync_mode": "overwrite"
+      "destination_sync_mode": "overwrite",
+      "primary_key": [["property_id"], ["cohort"], ["cohortNthDay"]]
     },
     {
       "stream": {
         "name": "pivot_report",
         "json_schema": {},
         "supported_sync_modes": ["full_refresh"],
-        "source_defined_primary_key": [["uuid"]]
+        "source_defined_primary_key": [["property_id"], ["browser"], ["country"], ["language"], ["date_range"]]
```

Comment:
Would it be more flexible for the destination to have the information as "start_date" and "end_date" since those can be cast as dates?

Reply:
changed